.github/workflows/secrets.yml

```diff
@@ -0,0 +1,24 @@
+name: Secrets and Environment Variables
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  top-secret:
+    runs-on: ubuntu-latest
+    env:
+      LIFERAY_RELEASE_GITHUB_PAT: ${{ secrets.LIFERAY_RELEASE_GITHUB_PAT }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Running test_tag_release test.
+        run: |
+         cd release         
+         output=$(./test_tag_release.sh)
+         echo "$output"
+         if [[ "$output" == *"Test failed."* ]]; then
+           echo "Test failed. Exiting with status 1."
+           exit 1
+         fi
+        shell: bash
```

Comment:
@lucasmiranda0, can you please try to sort as much as you can and format the code based on the current formatting we see in the release scripts?